src/MainWindow.cpp

```diff
@@ -78,7 +100,7 @@ void MainWindow::onAbout()
     aboutDialog.set_title("About");
     aboutDialog.set_version(VERSION);
     aboutDialog.set_program_name("whatsapp-for-linux");
-    aboutDialog.set_logo_icon_name("help-about");
+    aboutDialog.set_logo_icon_name("whatsapp-for-linux");
```

Comment:
Instead of this we should use set_default_icon(appIcon64x64) in the constructor because we cannot make sure our icon is installed or in the theme path in every environment. Setting the default icon takes care of about dialog icon as well.

Reply:
Can it be done like this?
```
auto const appIconList = get_icon_list();
aboutDialog.set_logo(appIconList[3]);
```

Reply:
Of course but the other way is more consistent I believe, as I said setting the default icon in constructor in `MainWindow` takes care of about dialog as well.

Reply:
Done https://github.com/eneshecan/whatsapp-for-linux/pull/49/commits/1c0f1311fd67afcc7fb4689ff4ab11eefec94755#diff-1c0b64226a129d35f02dc0049484cd6bR20